pkgs/development/python-modules/pysmartapp/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+, httpsig
+}:
+
+buildPythonPackage rec {
+  pname = "pysmartapp";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1i5f6as4f0jnba2zynni0v0y9fkf2qfavpyz73nk78as4j8ysnr9";
+  };
+
+  propagatedBuildInputs = [
+    six
+    httpsig
+  ];
+
+  doCheck = false;
```

Comment:
Why did you disable tests?

Reply:
The upstream repo contains [tests](https://github.com/andrewsayre/pysmartapp/tree/master/tests).